data/blog/m1sterc001guy-receives-lts-grant.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: "Long-Term Support For m1sterc001guy "
+date: '2024-11-11'
+tags: ['OpenSats', 'grants', 'bitcoin', 'LTS']
+draft: false
+authors: ['arvin', 'btcschellingpt', 'ecurrencyhodler']
+images: ['/static/images/blog/62-m1sterc001guy-lts.jpg']
+summary: "OpenSats welcomes m1sterc001guy as an LTS Grantee."
+---
+
+OpenSats is pleased to announce long-term support for
+[m1sterc001guy](https://github.com/m1sterc001guy/) for his continued
+contributions to the Bitcoin ecosystem.
+
+m1sterc001guy is the primary maintainer of the [Fedimint Lightning
+Gateway](https://fedibtc.github.io/fedi-docs/docs/gateways/intro/), playing an
+instrumental role in connecting federated communities to the wider Bitcoin
+Lightning network, enabling seamless payment experiences.
+
+As an [independent
+contributor](https://github.com/fedimint/fedimint/commits?author=m1sterc001guy&before=90a2fb72e8b52660911aea69919a8bdc1fa20ebc+70),
+he has made substantial enhancements that have strengthened Fedimint's
+functionality. His key improvements in database transactions, support for LND
+and LDK, and advancements in resilience, crash recovery, and backup systems have
+fortified Fedimint's security and accessibility for Bitcoin custody.
```

Comment:
nit suggestion: Would be cool to mention the ROASTr module here too (it is kind of background work for bullet point number 2 below). Maybe something like "He has also build an independent Fedimint module that allows federations to collaboratively sign Nostr notes"

https://github.com/m1sterc001guy/roastr